pages/integrations/azure.md

```diff
@@ -0,0 +1,81 @@
+# [Microsoft Azure IoT Suite][azure] integration
+
+[Resin.io][resin] makes it simple to deploy, update, and maintain code running on remote devices. Microsoft's new IoT Suite makes it easy manage and capture the data those devices generate. This tutorial will walk you through using the two in conjuction. 
+
+## Connect your device to resin.io
+
+Follow our [getting started guide][installing] or watch our screencast to get your device connect to your resin.io dashboard. Once your device shows up on the dashboard your ready for the next step.
+
+![factory-build](/img/integrations/azure/factory-build.png)
+
+## Create the resin.io API Key 
+
+Find your the Applications ID from the dashboard url: https://dashboard.resin.io/apps/NNNN/devices and your auth-token from the [preferences panel](https://dashboard.resin.io/preferences?tab=details). Then combine insert the two into the curl request below, it will return a permanent API key.
```

Comment:
unneeded "the" after "your"?. Also, "Application" in singular

"combine insert" also extra "combine"?

It must be noticed that the API key will be in quotes, but they should be stripped before pasting the key